client/my-sites/people/team-list/index.jsx

```diff
@@ -95,6 +112,7 @@ var Team = React.createClass( {
 		return (
 			<div>
 				<PeopleListSectionHeader
+					showRoles={ showRoles }
```

Comment:
Yes - similar to the changes to RoleSelect - this would arguably be better as a new component.  The prop here is just being used to switch on the select dropdown behaviour.